src/main/webapp/script.js

```diff
@@ -107,8 +108,19 @@ function createResultCard(name, address, photos, rating, totalCardCount) {
 
   const websiteButton = document.createElement('button');
   websiteButton.className = 'results-website-button';
-  websiteButton.innerText = 'Visit Website';
-  // TODO: Add website link
+  if (website.includes('maps.google.com')) {
+    websiteButton.innerText = 'Visit Website on Google Maps';
```

Comment:
I would make this "Visit Location on Google Maps"

Reply:
Changed!